tests/test_application.py

```diff
@@ -24,242 +24,253 @@ def GET(self):
 
 """
 
-urls = (
-    "/iter", "do_iter",
-)
+urls = ("/iter", "do_iter")
 app = web.application(urls, globals())
 
+
 class do_iter:
     def GET(self):
-        yield 'hello, '
-        yield web.input(name='world').name
+        yield "hello, "
+        yield web.input(name="world").name
 
     POST = GET
 
+
 def write(filename, data):
-    f = open(filename, 'w')
+    f = open(filename, "w")
     f.write(data)
     f.close()
 
+
 class ApplicationTest(unittest.TestCase):
     def test_reloader(self):
-        write('foo.py', data % dict(classname='a', output='a'))
+        write("foo.py", data % dict(classname="a", output="a"))
         import foo
+
         app = foo.app
 
-        self.assertEqual(app.request('/').data, b'a')
+        self.assertEqual(app.request("/").data, b"a")
 
         # test class change
         time.sleep(1)
-        write('foo.py', data % dict(classname='a', output='b'))
-        self.assertEqual(app.request('/').data, b'b')
+        write("foo.py", data % dict(classname="a", output="b"))
+        self.assertEqual(app.request("/").data, b"b")
 
         # test urls change
         time.sleep(1)
-        write('foo.py', data % dict(classname='c', output='c'))
-        self.assertEqual(app.request('/').data, b'c')
+        write("foo.py", data % dict(classname="c", output="c"))
+        self.assertEqual(app.request("/").data, b"c")
 
     def testUppercaseMethods(self):
         urls = ("/", "hello")
         app = web.application(urls, locals())
+
         class hello:
-            def GET(self): return "hello"
-            def internal(self): return "secret"
+            def GET(self):
+                return "hello"
 
-        response = app.request('/', method='internal')
-        self.assertEqual(response.status, '405 Method Not Allowed')
+            def internal(self):
+                return "secret"
+
+        response = app.request("/", method="internal")
+        self.assertEqual(response.status, "405 Method Not Allowed")
 
     def testRedirect(self):
         urls = (
```

Comment:
This format is really ugly.  :(

Reply:
The idea behind Black is to surrender your personal pet peeves in favor of a single standard.

Reply:
I understand the `surrender` part completely, what i mean is readability, not personal favor.

With `urls` like below is easier to read and understand IMO:

```
urls = (
    '<pattern>', 'class-or-func',
    '<pattern>', 'class-or-func',
    '<pattern>', 'class-or-func',
    '<pattern>', 'class-or-func',
    '<pattern>', 'class-or-func',
)
```

Reply:
Then let’s create a __namedtuple__ and write them that way.  The pattern, class_or_func thing is not obvious from reading the current code.